web-wallet/src/lib/components/Stake/Stake.svelte

```diff
@@ -191,7 +200,10 @@
 						limitUpper={gasSettings.gasLimitUpper}
 						price={gasSettings.gasPrice}
 						priceLower={gasSettings.gasPriceLower}
-						on:setGasSettings
+						on:setGasSettings={(event) => {
```

Comment:
This way you are creating the same function two times (here and at line 156).
It's better if you create a function inside the `<script/>` to be reused.